administrator/language/en-GB/com_guidedtours.ini

```
@@ -6,6 +6,10 @@
 COM_GUIDEDTOURS="Guided Tours"
 COM_GUIDEDTOURS_AUTOSTART_DESC="Start the tour automatically when a user reaches the context in which the tour should be displayed."
 COM_GUIDEDTOURS_AUTOSTART_LABEL="Auto Start"
+COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_DESCRIPTION="The amount of time (in seconds) a tour is delayed after being cancelled by the user and until it is shown again (only when a tour is set to start automatically).<br />For instance, enter 3600 for 1 hour, 86400 for 24 hours, 604800 for 1 week."
```

```suggestion
COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_DESCRIPTION="The amount of time (in seconds) a tour is delayed after being cancelled by the user and until it is shown again (only when a tour is set to start automatically).<br>For instance, enter 3600 for 1 hour, 86400 for 24 hours, 604800 for 1 week."
```

dont use xhtml formatting

---

All right, thanks Brian.